packages/react-theme/src/global/typeographyStyles.ts

```diff
@@ -0,0 +1,88 @@
+import { tokens } from '../tokens';
+import { TypeographyStyles } from '../types';
+
+/**
+ * Global typeography styles (fontSize, fontWeight, and lineHeight)
+ */
+export const typeographyStyles: TypeographyStyles = {
```

Comment:
`Text` already contains typography styles (which are not up to date): https://github.com/microsoft/fluentui/blob/5a6f0d55d87f70d67dd1567ce97e637e51b07863/packages/react-text/src/typographyStyles/typographyStyles.ts#L7.

We should remove one of them and keep just a single instance.

Reply:
I will do that update in a follow up PR immediately after this one is in.  Does that work?